bot/src/whitelist.ts

```diff
@@ -0,0 +1,21 @@
+import { getCollateralConfigByType } from 'auctions-core/src/constants/COLLATERALS';
+import { WHITELISTED_COLLATERALS } from './variables';
+
+const validateWhitelist = function (whitelist: string[]) {
+    whitelist.forEach(collateralType => {
+        getCollateralConfigByType(collateralType);
+    });
```

Comment:
the cherry on top would be to give user a list of collaterals that threw error, not just a first one (but that's not required to implement)

Reply:
We do like cherries! 

![Screenshot 2022-04-21 at 16 57 04](https://user-images.githubusercontent.com/30908158/164486454-d0d1f474-6d6f-46d6-b1ec-f9585918a6d8.png)